src/main/java/core/basesyntax/service/StorageServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageServiceImpl implements StorageService {
+    private final Map<String, Integer> storage = new HashMap<>();
```

Comment:
The encapsulation principle suggests that class fields should be private to hide the internal state and protect it from unauthorized access and modification. Consider changing the visibility of 'storage' to private.